tt_metal/impl/device/device.cpp

```diff
@@ -935,18 +936,19 @@ void Device::update_workers_build_settings(std::vector<std::vector<std::tuple<tt
                     compile_args[7]  = 0;
                     compile_args[8]  = dispatch_constants::get(dispatch_core_type).prefetch_q_size();
                     compile_args[9]  = CQ_PREFETCH_Q_RD_PTR;
-                    compile_args[10] = prefetch_d_settings.cb_start_address;
-                    compile_args[11] = prefetch_d_settings.cb_size_bytes;
-                    compile_args[12] = scratch_db_base;
-                    compile_args[13] = scratch_db_size;
-                    compile_args[14] = 0; //prefetch_sync_sem
-                    compile_args[15] = prefetch_d_settings.cb_pages; // prefetch_d only
-                    compile_args[16] = prefetch_d_settings.consumer_semaphore_id; // prefetch_d only
-                    compile_args[17] = demux_sem++; //prefetch_downstream_cb_sem, // prefetch_d only
-                    compile_args[18] = prefetch_d_settings.cb_log_page_size;
-                    compile_args[19] = dispatch_constants::PREFETCH_D_BUFFER_BLOCKS; // prefetch_d only
-                    compile_args[20] = true;  // is_dram_variant
-                    compile_args[21] = false; // is_host_variant
+                    compile_args[10] = CQ_PREFETCH_Q_PCIE_RD_PTR;
+                    compile_args[11] = prefetch_d_settings.cb_start_address;
+                    compile_args[12] = prefetch_d_settings.cb_size_bytes;
+                    compile_args[13] = scratch_db_base;
+                    compile_args[14] = scratch_db_size;
+                    compile_args[15] = 0; //prefetch_sync_sem
+                    compile_args[16] = prefetch_d_settings.cb_pages; // prefetch_d only
+                    compile_args[17] = prefetch_d_settings.consumer_semaphore_id; // prefetch_d only
+                    compile_args[18] = demux_sem++; //prefetch_downstream_cb_sem, // prefetch_d only
+                    compile_args[19] = prefetch_d_settings.cb_log_page_size;
+                    compile_args[20] = dispatch_constants::PREFETCH_D_BUFFER_BLOCKS; // prefetch_d only
+                    compile_args[21] = true;  // is_dram_variant
+                    compile_args[22] = false; // is_host_variant
```

Comment:
I know that this is just "shifting" the CT args down, but this setup path is a little janky, and I'm scared that this can easily break multi-chip functionality. Would you mind running the T3K tests before merging?

Reply:
Running it here to see: https://github.com/tenstorrent/tt-metal/actions/runs/10050307328

Reply:
Thank you!